src/lcdlib/lcdlib_common_oled.c

```diff
@@ -0,0 +1,531 @@
+#include <lcdlib/lcdlib_common_oled.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <linux/fb.h>
+
+// #define LCD_DEBUG
+#ifdef LCD_DEBUG
+#define print_err(fmt, args...)                                  \
+    do                                                           \
+    {                                                            \
+        FILE *log_file = fopen("/home/root/lcd_debug.log", "a"); \
```

Comment:
root folder is part SPI flash. Appending to it might fill partition.
Instead, we can use /tmp (ramdisk)

Reply:
ok, will do

Reply:
Can you please upload v2 to fix this?